packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -32,6 +33,7 @@ export interface ProfilePictureProps
 }
 
 export const sizeClasses: Record<ProfileImageSize, string> = {
+  size16: 'size-4',
```

Comment:
Maybe xxSmall to keep the same convention we have so far?

Reply:
yeah it's clearer this way, just until then it becomes a mess to have both.
Just wondering, since this is particular to this use case, wouldn't an override - custom size be better then defining?

This is more general discussions, since I see the sizing defined which should be finite, just grows with every usecase, but not a blocker for the PR.